libs/commons/Testutil.ml

```diff
@@ -20,4 +20,6 @@ let mask_temp_paths ?depth ?replace () =
       ~tmpdir:(UFilename.get_temp_dir_name () |> UUnix.realpath)
       ()
   in
+  (* NOTE: Haven't investigated why yet, but order is important here,
+   *       we must mask the physical path before the original path. *)
   fun text -> text |> mask_physical_path |> mask_original_path
```

Comment:
I believe this occurs when one path is a substring of the other. For example, the physical path is `/var/tmp` and the original path is `/tmp`. The string `/var/tmp/aaa` would be replaced incorrectly by `/var/<TMP>/<MASKED>`. This is because `Testo.mask_temp_paths` doesn't require the temp path to be at the beginning of the path. It should be fixable but maybe annoying because ocaml-re doesn't support lookbehind assertions.

Reply:
Here's the bug report: https://github.com/semgrep/testo/issues/55